packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/guides/Migration.tool.md

```diff
@@ -0,0 +1,519 @@
+
+### Accessibility Testing
+
+The dependency on [axe-core](https://github.com/dequelabs/axe-core) has been upgraded from v3.5.3 to v4.0.2. There were no new rules introduced, but previous rules had fixes added that might start detecting accessibility violations after the upgrade that were not previously being detected.
+
+Check out the [release notes](https://github.com/dequelabs/axe-core/releases) for more detail information about the changes.
+
+### Default Selector
+
+The default selector has been changed from `[data-terra-dev-site-content] *:first-child` to `[data-terra-test-content] *:first-child`. This selector is used as the default content region for capturing screenshots. This change should not affect most teams. For teams that are affected a custom selector can be provided using [service options](/dev_tools/terra-functional-testing/terra-functional-testing/wdio-services/terra-service#selector).
+
+## New Features
+
+### Accessibility Reporter
+
+A custom WebDriverIO reporter has been created to facilitate monitoring accessibility violations and reporting them upon the completion of the test run. The goal of this reporter is to allow us to more frequently upgrade our version of axe-core without as many breaking changes. When axe adds new rules those rules can be marked as warnings and the accessibility reporter will report them to the terminal without failing as a violation. Please monitor your terminal output to stay proactive on accessibility warnings.
+
+### Terra CLI
+
+Our command line interface has been transitioned to [Terra CLI](https://engineering.cerner.com/terra-ui/dev_tools/cerner/terra-cli/about) to create a uniform experience across our scripts. All new commands are integrated plugins for the terra-cli. Command options are constructed using [yargs](https://github.com/yargs/yargs);
+
+As part of this upgrade teams will be updating the scripts used to invoke wdio to use the terra-cli commands. The `terra wdio` script will invoke the test runner.
+
+Note: The terra cli array parameters have a different syntax. [Array](https://github.com/yargs/yargs/blob/master/docs/api.md#array) items are now delimited by a space.
+
+```diff
+// package.json
+{
+  "scripts": {
+-    "test:wdio": "wdio"
++    "test:wdio": "terra wdio"
+-    "test:wdio:local": "tt-wdio --gridUrl='grid.test.example.com' --locales=['de','en-AU'] --browsers=['chrome','firefox','ie']"
++    "test:wdio:local": "terra wdio --gridUrl grid.test.example.com --locales de en-AU --browsers chrome firefox ie"
+  }
+}
+```
+
+### Auto Update Screenshots
+
+A CLI option has been added to auto-update reference screenshots during the test run. When enabled all screenshots generated during the test run will automatically overwrite the existing reference screenshot. Be sure to manually validate screenshot updates when using this feature.
+
+```sh
+npm run terra wdio --updateScreenshots
```

Comment:
Should these just be `terra wdio --updateScreenshots` without `npm run` or are we expecting that they also create a `terra` script?

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/574/commits/fb337feddb2c24866952c8a5f2ac8b7f5320b5b4